.gitignore

```
@@ -5,6 +5,7 @@ target
 .direnv
 
 *.zip
+target_pol*
```

The trailing `*` is unnecessary here. `target_pol*` means any file or directory that starts with `target_pol`. So this would also ignore `target_pol_foobar`, but only directories named `target_pol` should be ignored.
```suggestion
target_pol
```